README.md

```diff
@@ -78,17 +78,33 @@ A [Bonsai](https://bonsai-rx.org/) workflow for lick-based foraging experiments,
 - Copy `Settings_box<box num>.csv` to `Users\svc_aind_behavior\Documents\ForagingSettings`
   - Copy `<box num>` 1-4 depending on the computer
   - Configure the COM ports for each computer
+    - Behavior: 
+        - How are these determined?
+    - Soundcard:
+        - How are these determined?
```

Comment:
We first have a physical mapping to determine which USB ports these harp devices should use. We then set the COM port for that physical port so that we get the same COM port for that physical port on all computers.